agent/stats/queue_test.go

```diff
@@ -309,3 +309,64 @@ func TestCpuStatsSetNotSetToInfinity(t *testing.T) {
 		t.Errorf("Computed cpuStatsSet.SampleCount (%d) != expected value (%d)", sampleCount, 1)
 	}
 }
+
+func TestResetThreshholdElapsed(t *testing.T) {
+	// create a queue
+	queueLength := 3
+	queue := NewQueue(queueLength)
+
+	queue.Reset()
+
+	threshholdElapsed := queue.resetThreshholdElapsed(2 * time.Millisecond)
+	if threshholdElapsed {
+		t.Errorf("Queue is NOT expected to elapse the threshhold right after reset")
+	}
+
+	time.Sleep(3 * time.Millisecond)
+	threshholdElapsed = queue.resetThreshholdElapsed(2 * time.Millisecond)
+
+	if !threshholdElapsed {
```

Comment:
Same here. Please use the `github.com/stretchr/testify/assert` package's `True()` method here as well.